### ISSUE TITLE
Add state = 4 in getChannelInfo (settled, but pending unlocks)

### ISSUE THREAD

**Issue by loredanacirstea** [raiden-network/raiden-contracts#211]
As discussed in:
https://github.com/raiden-network/raiden-contracts/pull/198#discussion_r206953616

and
>  I think it can be a good idea to also add the participant addresses and have the state for the unlock pending, additionally we should know which of the participants is pending unlock.
https://github.com/raiden-network/raiden-contracts/pull/198#pullrequestreview-142489240

If https://github.com/raiden-network/raiden-contracts/pull/198 and https://github.com/raiden-network/raiden-contracts/pull/203 are merged, we can add an additional channel state in `getChannelInfo` to have all the channel states through only one endpoint:

`function getChannelInfo(uint256 channel_identifier, address participant1, address participant2)`

The participant address are needed, so the smart contract can also retrieved the stored unlock data for both participants.

**Comment by loredanacirstea** [comment]
@hackaugusto , is the following enough:
also @LefterisJP 

tl;dr
Get actual channel `state` with `getChannelInfo`. If you want to see what participant has `unlock` data on chain, use `getChannelParticipantInfo` and check ` locksroot` & `locked_amount`.

```js
function getChannelInfo(uint256 channel_identifier, address participant1, address participant2)
        view
        external
        returns (uint256, uint8)
{
        // returns (settle_block_number, state)
        // where state can be:
        //  0 = non-existent
        //  1 = open
        //  2 = closed
        //  3 = settled
        //  4 = pending_unlock (at least 1 unlock)
}

function getChannelParticipantInfo(
            uint256 channel_identifier,
            address participant,
            address partner
)
        view
        external
        returns (uint256, uint256, bool, bytes32, uint256, bytes32, uint256)
{
        // returns (deposit, withdrawn_amount, is_the_closer, balance_hash, nonce, locksroot, locked_amount)
}

```

https://github.com/raiden-network/raiden-contracts/pull/203 already implements the above `getChannelParticipantInfo`
